includes/Modules/Analytics_4.php

```diff
@@ -493,11 +534,27 @@ function( $property ) {
 				return array_map( array( self::class, 'filter_webdatastream_with_ids' ), $webdatastreams );
 			case 'GET:webdatastreams-batch':
 				return self::parse_webdatastreams_batch( $response );
+			case 'GET:container-lookup':
+				return (array) $response;
+			case 'GET:container-destinations':
+				return (array) $response->destination;
```

Comment:
Please use the respective methods instead. Here `$response` is a `\Google\Site_Kit_Dependencies\Google\Service\TagManager\ListDestinationsResponse` so we should use `getDestination`.

```suggestion
				return (array) $response->getDestination();
```

This is okay to cast to an array as it will likely be `null` otherwise for a tag with no destinations.

Reply:
Ah thanks, updated.

Reply:
Is your editor maybe configured to ignore the `/third-party` directory? 
<img width="978" alt="image" src="https://user-images.githubusercontent.com/1621608/209217390-50efd035-d68c-4b14-a529-df8b72aceb21.png">